README.md

```diff
@@ -1 +1,111 @@
-Coming soon...
+# `gpii-binder`
+
+This package provides a mechanism for binding [Fluid view component](http://docs.fluidproject.org/infusion/development/tutorial-gettingStartedWithInfusion/ViewComponents.html) model variables to DOM elements using [selectors](http://docs.fluidproject.org/infusion/development/tutorial-gettingStartedWithInfusion/ViewComponents.html#selectors).
+
+You can bind to any DOM element whose value can be read and set using
+[`fluid.value`](http://docs.fluidproject.org/infusion/development/ViewAPI.html#fluid-value-nodein-newvalue-), but the
+primary (and tested) use case is binding model variables to form elements, specifically:
+
+* text `<input>` fields
+* radio `<input>` fields
+* checkbox `<input>` fields
+* `<textarea>` fields
+* `<select>` fields
+
+Once you run `gpii.binder.applyBinding(component)` (see "Static Functions" below), a binding is created
+between any selectors and model variables referenced in`options.binding` (see "Supported options" for the format).
+
+Once a binding exists, changes to a bound model sent using [the change applier](http://docs.fluidproject.org/infusion/development/ChangeApplier.html)
+are used to update the DOM element's value.
+
+The binding is bidirectional.  Change events to a bound DOM element's value are also relayed to the associated model
+variable.  Note that change events are not generated when you directly set the element's value, but only when you have
+updated the value using browser events *and* change focus.  For more details, see the
+[jQuery documentation for the change event](https://api.jquery.com/change/).
+
+
+# Supported options
+
+The `gpii.binder.applyBinding` function provided by this package can only do its work if you have the
+following options defined:
+
+| Option             | Type     | Description |
+| ------------------ | -------- | ----------- |
+| `selectors` | `{Object}` | You must define one or more [selectors](http://docs.fluidproject.org/infusion/development/tutorial-gettingStartedWithInfusion/ViewComponents.html#selectors) that can be used in a binding. |
+| `bindings` | `{Object}` | Defines the relationship between selectors and model variables.  The full notation for this option is outlined below. |
+
+## Long notation
+
+There are two ways of specifying bindings.  The "long form" has named keys (as in the first example above) and
+supports the following options:
+
+* selector: A valid selector for your component.  Must be able to be resolved using `that.locate(selector)`
+* path: A valid path for the model variable whose value will be watched.  Must be able to be resolved using `fluid.get(path)`.
```

Comment:
More specifically, fluid.get(that.model, path) where "that" is the bound component